packages/account/src/tests/test_account.cairo

```diff
@@ -50,7 +52,7 @@ fn setup(key_pair: StarkKeyPair) -> ComponentState {
 fn setup_dispatcher(
     key_pair: StarkKeyPair, data: SignedTransactionData
 ) -> (AccountABIDispatcher, felt252) {
-    let contract_class = declare("DualCaseAccountMock").unwrap_syscall();
+    let contract_class = *declare("DualCaseAccountMock").unwrap_syscall().contract_class();
```

Comment:
We could use `utils::declare_class` here, that would make code a bit nicer and we wouldn't have to import `DeclareResultTrait`